packages/webapp/src/containers/Home/index.jsx

```diff
@@ -39,7 +42,7 @@ export default function Home({ history }) {
     <PureHome
       greeting={t('HOME.GREETING')}
       first_name={userFarm?.first_name}
-      imgUrl={userFarm.farm_image_url || imgUrl}
+      imgUrl={authenticatedImageUrl || (isLoading ? '' : defaultImageUrl)}
```

Comment:
The isLoading check ensures it remains a white background until the authenticatedImageUrl is set. Otherwise it would flicker with the defaultImage